CORE
main.py
--force-malloc-success --unwind 10
^VERIFICATION SUCCESSFUL$
